### ISSUE TITLE
SEO/Accessibility: Add alt text to GIzra logo [1h]

### ISSUE THREAD

**Issue by noamoss** [Gizra/gizra.com#660]
Gizra Logo image should have a descriptive alt text: "Gizra, We Build Websites".

**Comment by IshaDakota** [comment]
Merged & deployed
